### PR TITLE
Backend

### PR DESCRIPTION
**Pull request by Kobzol**

### CODE REVIEW

File: elsie/utils/cache.py [hunk 1]
```diff
@@ -4,11 +4,10 @@
 
 
 class FsCache:
-    def __init__(self, cache_dir, version, inkscape_version):
-        start = ("elsie-{}/{}/".format(version, str(inkscape_version))).encode()
+    def __init__(self, cache_dir, version):
```
Comment (spirali) [inline elsie/utils/cache.py:7]: Inkscape version should be there, otherwise cache may be corrupted

File: elsie/slides/slidedeck.py [hunk 2]
```diff
@@ -405,25 +327,29 @@ def render(
             slide_postprocessing([slide.box() for slide in select_slides])
 
         if prune_cache:
-            self.query_cache = self.used_query_cache
-            self.used_query_cache = {}
+            self.backend.prune_cache()
 
         if save_cache:
-            self._save_query_cache(self.query_cache)
```
Comment (spirali) [inline elsie/slides/slidedeck.py:412]: query cache is somehow independent on the backend, more precisely it stays over the backends, so saving the case should not probably be part of the backend. But I will have to think about it more.

File: elsie/utils/cache.py (hunk 1 above)
Comment (Kobzol) [reply elsie/utils/cache.py:7]: It is, see slidedeck.py:128 (https://github.com/spirali/elsie/pull/70/files#diff-4b8678483ad0b238be36aff422cdc12f8b4d6083e61302eea465bbadc09b1970R108).

Comment (spirali) [reply elsie/utils/cache.py:7]: Ah I see, the version already contains a the Inkscape version

**Pull request merged**